modular_skyrat/modules/assaultops/code/gamemode/assaultops.dm

```diff
@@ -0,0 +1,271 @@
+
+	var/command_radio = FALSE
+	var/cqc = FALSE
+
+/datum/outfit/assaultops/cqb
+	name = "Assault Operative - CQB"
+
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/c20r,\
+		/obj/item/ammo_box/magazine/smgm45=4,\
+		)
+
+	cqc = TRUE
+
+/datum/outfit/assaultops/demoman
+	name = "Assault Operative - Demolitions"
+
+	belt = /obj/item/storage/belt/grenade/full
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/gyropistol,\
+		/obj/item/ammo_box/magazine/m75=4,\
+		/obj/item/implant/explosive/macro, \
+		/obj/item/storage/box/assaultops/demoman
+		)
+	l_hand = /obj/item/clothing/suit/space/hardsuit/rd
+
+/obj/item/storage/box/assaultops/demoman
+	name = "Assault Operative - Demolitions"
+
+/obj/item/storage/box/assaultops/demoman/PopulateContents()
+	for(var/i in 1 to 4)
+		new /obj/item/grenade/syndieminibomb(src)
+		new /obj/item/grenade/c4(src)
+
+/datum/outfit/assaultops/medic
+	name = "Assault Operative - Medic"
+
+	glasses = /obj/item/clothing/glasses/hud/health
+	belt = /obj/item/storage/belt/medical/paramedic
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/submachine_gun/pps,\
+		/obj/item/ammo_box/magazine/pps=4,\
+		/obj/item/storage/firstaid/tactical=2,\
+		/obj/item/gun/medbeam)
+
+/datum/outfit/assaultops/heavy
+	name = "Assault Operative - Heavy Gunner"
+
+	suit = /obj/item/clothing/suit/space/hardsuit/syndi/elite
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/l6_saw/unrestricted/mg34/assaultops,\
+		/obj/item/ammo_box/magazine/mg34=4,\
+		/obj/item/grenade/syndieminibomb)
+
+/obj/item/gun/ballistic/automatic/l6_saw/unrestricted/mg34/assaultops
+	w_class = WEIGHT_CLASS_NORMAL
+
+/datum/outfit/assaultops/assault
+	name = "Assault Operative - Assault"
+
+	suit = /obj/item/clothing/suit/space/hardsuit/syndi/elite
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/assault_rifle/akm,\
+		/obj/item/ammo_box/magazine/akm=4,\
+		/obj/item/grenade/syndieminibomb=2)
+
+/datum/outfit/assaultops/sniper
+	name = "Assault Operative - Sniper"
+
+	suit = /obj/item/clothing/suit/space/hardsuit/syndi/elite
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/kitchen/knife/combat/survival,\
+		/obj/item/gun/ballistic/automatic/sniper_rifle/syndicate,\
+		/obj/item/ammo_box/magazine/sniper_rounds=4,\
+		/obj/item/grenade/syndieminibomb=2)
+
+/datum/outfit/assaultops/tech
+	name = "Assault Operative - Tech"
+
+	suit = /obj/item/clothing/suit/space/hardsuit/shielded
+
+	belt = /obj/item/storage/belt/military/abductor/full
```

Comment:
Yeah, We all know that the syndicate collaborate with abductors, right?
No, But seriously. It would be nicer to just get a normal toolbelt
It basically give them a welding tool with infinite fuel, Multitool that can see which wires do which and increased action speeds